lib/components/Select/Select.styled.ts

```diff
@@ -50,13 +52,17 @@ export const SelectItems = styled.div<{
     display: ${({ focus }) => (focus ? "flex" : "none")};
     position: absolute;
     z-index: 99999999;
-    top: ${({ theme, description, helper }) =>
-      `calc(
+    top: ${({ theme, description, helper, expandUp, menuHeight }) =>
```

Comment:
in expandUp cases, you should ommit the 'top' rule and style it using the 'bottom' one for consistency. if you use bottom you can ditch the whole calculation below and just say "bottom: 100px" and it will work.